book/styles.md

```diff
@@ -68,45 +78,50 @@ class BlockLayout:
         # ...
 ```
 
-Put the exact same lines of code inside `InlineLayout` as well, so that
-paragraphs and list items and so on can have backgrounds as well. For now, also
-remove the default gray background from `pre` elements, but we'll put it back
-soon.
+Put the exact same lines of code inside `InlineLayout` as well, so
```

Comment:
Your code above uses a static method and brand checking on class names instead...

Reply:
Note sure what you mean here.

Reply:
Looks like my comment is outdated anyway, so disregard.